runtime_config_linux.go

```diff
@@ -182,7 +182,9 @@ type CPU struct {
 
 // Pids for Linux cgroup 'pids' resource management (Linux 4.3)
 type Pids struct {
-	// Maximum number of PIDs. A value < 0 implies "no limit".
+	// Maximum number of PIDs. A value <= 0 implies "no limit".
+	// While a limit of 0 is technically valid, it is precisely identical to
+	// a limit of 1, so there is no distinction.
```

Comment:
On Wed, Oct 28, 2015 at 09:28:23AM -0700, Aleksa Sarai wrote:
> Oh, sorry. What I meant was that "<= 0 means no limit *within the
> spec*, but a limit of 0 is identical to a limit of 1 in the PIDs
> controller". I guess I'm mixing specification and justification,
> based on the discussion in opencontainers/runc#58.

Oh, I see.  In that case, my personal preference would be to just use
a uint pointer with “not set in the JSON means the runtime should
ignore” (see discussion in #233).  But however this lands, any
semantic rules (like what values the runtime should ignore) should
also land in the runtime-config-linux.md docs where bundle authors
will find them.